src/Ast.ml

```diff
@@ -140,8 +140,8 @@ let has_trailing_attributes_exp {pexp_desc; pexp_attributes} =
       false
   | _ ->
       List.exists pexp_attributes ~f:(function
-        | {Location.txt= "ocaml.doc" | "ocaml.text"}, _ -> false
-        | _ -> true )
```

Comment:
So that I understand what is desired, in this case the red formatting is preferred, right?

Reply:
Yes